core/src/main/java/com/github/kaktushose/jda/commands/dispatching/handling/ComponentHandler.java

```
@@ -36,8 +37,7 @@ protected InvocationContext<GenericComponentInteractionCreateEvent> prepare(Gene
             case StringSelectInteractionEvent event -> new ArrayList<>(List.of(event.getValues()));
             case EntitySelectInteractionEvent event -> new ArrayList<>(List.of(event.getMentions()));
             case ButtonInteractionEvent _ -> new ArrayList<>();
-            default ->
-                    throw new IllegalStateException("Should not occur. Please report this error to the devs of jda-commands.");
+            default -> throw new JDACException.Internal("Should not occur. Please report this error to the devs of jda-commands.");
```

"Please report this error to the devs of jda-commands." can be removed, would be duplicated